### ISSUE TITLE
home link broken (minor)

### ISSUE THREAD

**Issue by pirate486743186** [piotrex/XOWA-Addon#3]
The link to home in the menu and in the main page is broken.
it tries to open "xowa://home/wiki/"
Copy pasting the link "xowa://home/wiki/Main_Page" in the url bar works fine

**Comment by gnosygnu** [comment]
I was able to reproduce the sidebar links failing. I'll get that fixed for v2.7.1

However, I wasn't able to get the logo to fail. Can you try to reproduce with my steps below? If your steps are different, please detail and I'll try to reproduce on my side.

Thanks.

* Launch XOWA in --app_mode server
* Clicked on Help:License/Program
* Clicked on the logo -> redirected back to main page

**Comment by pirate486743186** [comment]
Not the image of the logo, i meant the link in the logo in help/home.

But it seams i was wrong a bit.
this works
"xowa://simple.wikipedia.org/wiki/"

mysteriously "xowa://home/wiki/" (the link in the logo) doesn't work.
It just says "Error"

this works "xowa://home/wiki/Main_Page" (i thought the links had to be like this)

The link in the tool bar, tries to launch "xowa:home" and fails.

**Comment by gnosygnu** [comment]
Hmm... That's strange.

I definitely agree with your #2. The wiki links in the sidebar are broken.

However I can't reproduce #1 or #3.

* The link in the extension menu / toolbar works 
* Clicking the XOWA logo works
* Entering in "xowa://home/wiki/" also works

I'm using the latest version of XOWA v2.6.5 on a Windows 64 bit machine with Firefox 38.0.5.

Out of curiosity, are you using the latest XOWA? If you are, what type of machine / firefox do you have?

**Comment by pirate486743186** [comment]
I'm on linux, firefox 38 and 2.6.4.1 xowa.
It's probably a Linux only bug/specificity.

I start to be really confused about what should or shouldn't work or all the very similar links.

In firefox 

xowa:simple.wikipedia.org/" works
The links in the sidebar try to open "xowa://simple.wikipedia.org/" and fail.
All the other combinations work.

"xowa://home" and "xowa:home" fail
"xowa://home/wiki/" and "xowa:home/wiki/" fail
all other combinations work

in the linux build in browser

"home/wiki/" and "home" don't work. 
But you don't see that problem, because you don't have these links there.

Your test works in the build in browser, but the link of the logo is "home/Main_Page"(that link works and send you to "home/wiki/Main_Page").

Also on start up you arrive at "home/wiki/Main_Page"

Also you have "simple.wikipedia.org/Main_Page" for the wikis.
But "simple.wikipedia.org/wiki/"  and "simple.wikipedia.org/" do work. They send you to "simple.wikipedia.org/wiki/Main_Page"

However, trying to open a new tab with the logo doesn't work (just a white page).

In short, in the build in browser in Linux.
1. "home/wiki/" and "home" are the only urls that don't work. But they aren't a problem there, because they aren't used anywhere.
2. Opening a new tab from the logo doesn't work
3. all the possible links for the wikis work

**Comment by gnosygnu** [comment]
> I'm on linux, firefox 38 and 2.6.4.1 xowa.
> It's probably a Linux only bug/specificity.

Cool. Thanks for confirming. You're right it is a Linux-only bug. I'm able to reproduce it on my Linux box, but not my Windows box.

I'll look at it later, but it'll take me some time.

> "home/wiki/" and "home" don't work.

Yup, in the GUI, both of these don't work. "home" is a special wiki, and I didn't want to intercept the requests. For example, if you're in English Wikipedia and type "home" you should go to "en.wikipedia.org/wiki/home", not "home/wiki/Main_Page"

> but the link of the logo is "home/Main_Page" (that link works and send you to "home/wiki/Main_Page")

Yeah, that's an inconsistency on my side. "home/wiki/Main_Page" is the correct standard. "home/Main_Page" is deprecated. Same for "simple.wikipedia.org/Main_Page" I'll remove it in a future build

> However, trying to open a new tab with the logo doesn't work (just a white page).

Cool. Thanks for reporting! I'll fix that for next release (or the one afterwards)

**Comment by gnosygnu** [comment]
I've fixed the following for v2.7.1 (to be released sometime Sunday evening)

* "XOWA Home" in toolbar now works on Firefox Linux
* Wiki links in sidebar works on Firefox (all OS)
* Clicking on logo opens up page on Firefox Linux
* Middle-clicking on logo opens up new tab in GUI (all OS)
* Status bar now shows "home/wiki/Main_Page" instead of "home/Main_Page". I had actually chosen the latter deliberately years ago (because it was shorter), but realize now that the former is more standard.

I think that's everything for this issue. Let me know if I missed anything.

Finally, I like to credit users for discovering bugs in the XOWA change log. Right now, there are a few lines like "Fix middle-click on logo not working {detected by anonymous}". Let me know if you'd rather go by pirate486743186 or would rather remain as anonymous.

Thanks!

**Comment by pirate486743186** [comment]
lol change it as pirate

**Comment by gnosygnu** [comment]
Cool. "pirate" it is.

@piotrex : Feel free to close out this issue whenever. Thanks!

**Comment by pirate486743186** [comment]
"XOWA Home" in toolbar doesn't work.
It tries to open "xowa:home" B[
This is with the addon bundled with xowa_app_linux_64_v2.8.3.1
upgrading the addon, brakes it completely, see other issue.

**Comment by gnosygnu** [comment]
Sorry about that. I broke this in v2.8.1 when I rewrote a lot of the URL logic.

I'll fix the xowa:home link from the toolbar for v2.8.4.

Thanks.

**Comment by gnosygnu** [comment]
Just confirming that this both the toolbar and the sidebar links are fixed in tonight's release. Thanks for reporting it.
